certbot/main.py

```diff
@@ -68,17 +70,17 @@ def _report_successful_dry_run(config):
                                   reporter_util.HIGH_PRIORITY, on_crash=False)
 
 
-def _auth_from_domains(le_client, config, domains, lineage=None):
+def _auth_from_available(le_client, config, domains=None, certname=None, lineage=None):
     """Authenticate and enroll certificate.
```

Comment:
Ha yes. Especially given that this is the primary `main` authentication method, as far as I can tell.